### ISSUE TITLE
feature request:  Automatically open panel when command start running.

### ISSUE THREAD

**Issue by scippio** [stevearc/overseer.nvim#318]
### Did you check existing requests?

- [X] I have searched the existing issues

### Describe the feature

Automatically open panel when command start running.

### What is the significance of this feature?

nice to have

**Comment by stevearc** [comment]
You should add the [open_output](https://github.com/stevearc/overseer.nvim/blob/master/doc/components.md#open_output) component to your task; that will open the Overseer panel when the task starts. You can put it in the default list of components added to all tasks, or use `add_template_hook` to add it to certain tasks, or manually add it after creating but before starting the task

**Comment by scippio** [comment]
Sorry the documentation is not so clear... I just want to open task list on the right after task started...

**Comment by nishantpillai5** [comment]
+1
